app/apply/page.tsx

```diff
@@ -0,0 +1,831 @@
+              className={`${baseInput} ${errors.semester && touched.semester ? errorInput : ""}`}
+              required
+            >
+              <option value="">Select Semester</option>
+              <option value="Semester 1">Semester 1</option>
+              <option value="Semester 2">Semester 2</option>
+            </select>
+            {errors.semester && touched.semester && (
+              <p className="mt-1 text-xs text-red-600">{errors.semester}</p>
+            )}
+          </div>
+        </div>
+
+        {/* Specialization */}
+        <div>
+          <label className="block text-sm font-medium mb-1">Specialization *</label>
+          <select
+            name="specialization"
+            value={specialization}
+            onChange={(e) => setSpecialization(e.target.value)}
+            onBlur={() => handleBlur("specialization")}
+            className={`${baseInput} ${errors.specialization && touched.specialization ? errorInput : ""}`}
+            required
+          >
+            <option value="">Select Specialization</option>
+            <option value="Software Engineering">Software Engineering</option>
+            <option value="Data Science">Data Science</option>
+            <option value="Computer Science">Computer Science</option>
+            <option value="Information Technology">Information Technology</option>
+            <option value="Interactive Media">Interactive Media</option>
+            <option value="Artificial Intelligence">Artificial Intelligence</option>
+            <option value="Information Systems Engineering">Information Systems Engineering</option>
+            <option value="Cyber Security">Cyber Security</option>
+            <option value="Computer Systems Engineering">Computer Systems Engineering</option>
+            <option value="Other">Other</option>
+          </select>
+          {errors.specialization && touched.specialization && (
+            <p className="mt-1 text-xs text-red-600">{errors.specialization}</p>
+          )}
+        </div>
+
+        {/* WhatsApp Split with Country Code Datalist */}
+        <div>
+          <label className="block text-sm font-medium mb-1">WhatsApp Number *</label>
+          <div className="flex flex-col sm:flex-row gap-3">
+            <div className="flex items-stretch">
+              <span className="px-2 py-2 border border-r-0 rounded-l bg-gray-50 text-sm select-none">
+                +
+              </span>
+              <input
+                name="whatsappCountry"
+                list="country-codes"
+                value={whatsappCountryDigits}
+                onChange={(e) => {
+                  const raw = e.target.value
+                  const digitsMatch = raw.match(/^\d{1,3}/)
+                  const digits = digitsMatch ? digitsMatch[0] : raw.replace(/[^0-9]/g, "")
+                  setWhatsappCountryDigits(digits.slice(0, 3))
+                  setFieldTouched("whatsappCountry")
```

Comment:
Simpler way to store only digits from user input.

```suggestion
                  const digits = e.target.value.replace(/\D/g, "").slice(0, 3)
                  setWhatsappCountryDigits(digits)
```